src/site/paragraphs/table.drupal.liquid

```diff
@@ -55,6 +55,8 @@
                         {% endfor %}
                     </tr>
             {% endif %}
+            {% if forloop.last %}
                 </tbody>
+            {% endif %}
```

Comment:
This was actually a separate issue that I learned about in passing once...We were add a new closing `tbody` for each table row